tests/dtm/test_rgw_s3_process_restart.py

```diff
@@ -756,6 +771,7 @@ def test_overwrite_same_object_during_rgw_restart(self):
                                                           max_object_size, que))
         proc_overwrite_op.start()
 
+        time.sleep(self.delay)
```

Comment:
same delay needed at all places?

Reply:
Yes, added the delay for all 'IO during restart service'. 
As s3bench takes few seconds for starting actual IO operations.